core/src/main/scala/com/softwaremill/react/kafka/ReactiveKafka.scala

```diff
@@ -95,15 +98,47 @@ class ReactiveKafka {
     Sink.fromGraph(new KafkaGraphStageSink(producer))
   }
 
+  // Scala DSL
+  def graphStageSource[K, V](
+    props: ConsumerProperties[K, V],
+    topicsAndPartitions: List[TopicPartition] = List(),
```

Comment:
One more thing: I think it's better to keep two separate functions: One that takes a `List[TopicPartition]` and one that takes a `List[TopicPartitionOffset]`.